README.md

```diff
@@ -1,9 +1,10 @@
-#An Unexpected Surprise<img width="25%" align="right" src="http://i.imgur.com/wfamleY.jpg">
```

Comment:
This is so weird, even for when I wrote this I wouldn't have looked at that and felt okay. Oof.